Dheavyman/react-redux-project
# Add ajax request status update and error handling

#### What does this PR do?
- Add functionality update ajax status and handle promise failure for async actions
#### Description of Task to be completed?
- Add a status update to the application when saving a new course
- Add error handling for failed ajax requests
#### How should this be manually tested?
- Pull and checkout to the branch with the command: `git fetch async-status-error-handling:async-status-error-handling` `git checkout async-status-error-handling`
- Start the application with the command: `npm start`. The application will open on the default browser
- Click on courses to go to the courses page
- Click on add courses button to add a new course
- Click on the save button without filling the form to see the error handling
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- N/A
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
